openstack/glance/templates/seed.yaml

```diff
@@ -23,13 +23,13 @@ spec:
     endpoints:
     - interface: admin
       region: {{ .Values.global.region | quote }}
-      url: '{{ .Values.services.api.scheme }}://{{ .Values.services.api.host }}:{{ .Values.services.api.port }}'
+      url: http://{{include "glance_api_endpoint_host_admin" .}}:{{.Values.api_port_internal}}
     - interface: public
       region: {{ .Values.global.region | quote }}
-      url: '{{ .Values.services.api.scheme }}://{{ .Values.services.api.host }}:{{ .Values.services.api.port }}'
+      url: https://{{include "glance_api_endpoint_host_public" .}}:{{.Values.api_port_internal}}
```

Comment:
no need for the port, we take the default 443

Reply:
would i need to remove {{.Values.api_port_internal}}? it be like:

url: https://{{include "glance_api_endpoint_host_public" .}}

or 

change api_port_internal value to 443 in values.yaml ?

Reply:
removing the api_port_internal here, like done for barbican